app/src/main/java/de/tum/in/tumcampusapp/component/ui/overview/InformationActivity.kt

```diff
@@ -55,16 +56,24 @@ class InformationActivity : BaseActivity(R.layout.activity_information) {
         }
     }
 
+    /**
+     * Open the Facebook app or view page in a browser if Facebook is not installed
+     */
     private fun openFacebook() {
-        // Open the Facebook app or view page in a browser if Facebook is not installed
+        val default = Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.facebook_link)))
         val facebook = try {
             packageManager.getPackageInfo("com.facebook.katana", 0)
             Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.facebook_link_app)))
-        } catch (e: PackageManager.NameNotFoundException) {
-            Intent(Intent.ACTION_VIEW, Uri.parse(getString(R.string.facebook_link)))
+        } catch (e: Exception) {
+            default
         }
```

Comment:
Assignment seems overly complicated. You can just have `startActivity` inside the try block here and save on that extra logic.